src/plugin-examples/plugin-examples.stories.tsx

```diff
@@ -1,23 +1,24 @@
-import React, { useCallback, useRef, useState } from "react";
-import { Value } from "slate";
+import React, { useCallback, useMemo } from "react";
 import IconVariable from "./icon-variable";
-import { textToSlate } from "../common/slate-types";
-import { kVariableFormatCode, VariablesPlugin } from "./variable-plugin";
+import { EFormat, textToSlate } from "../common/slate-types";
+import { kVariableFormat} from "./variable-plugin";
 import { IProps as ISlateToolbarProps, SlateToolbar, ToolbarTransform } from "../slate-toolbar/slate-toolbar";
 import { getPlatformTooltip, IButtonSpec } from "../editor-toolbar/editor-toolbar";
-import { IProps as ISlateEditorProps, SlateEditor } from "../slate-editor/slate-editor";
-import { Editor } from "slate-react";
+import { IProps as ISlateEditorProps } from "../slate-editor/slate-editor";
+import { Editable, Slate, withReact } from "slate-react";
+import { withHistory } from "slate-history";
+import { createEditor } from "../common/create-editor";
 
 export default {
   title: "Plugin Examples"
 };
 
 /*
- * Variables
+ * Variables: FIXME: put this story back in
```

Comment:
I originally got this working by cheating and not making it a full plugin. e.g. I defined the variable in the `EFormat` enum and added it directly to createEditor the same way that links were done. I did get this working in CLUE without doing that, I just didn't have time to come back and update this.  The CLUE version lives in variables-plugin on the slate-stuff branch. The draft PR is here:  https://github.com/concord-consortium/collaborative-learning/pull/1525